xds/src/main/java/io/grpc/xds/client/XdsLogger.java

```diff
@@ -14,9 +14,10 @@
  * limitations under the License.
  */
 
-package io.grpc.xds;
+package io.grpc.xds.client;
```

Comment:
We have documentation and code referencing this class to enable logging. We're going to need to do this carefully or we tweak how we use it.